integration/setup/common.go

```diff
@@ -44,48 +49,149 @@ func common(ctx context.Context, config Config) error {
 	}
 
 	{
-		c := chartvalues.CertOperatorConfig{
-			CommonDomain:       env.CommonDomain(),
-			RegistryPullSecret: env.RegistryPullSecret(),
-			Vault: chartvalues.CertOperatorVault{
-				Token: env.VaultToken(),
-			},
-		}
-
-		values, err := chartvalues.NewCertOperator(c)
+		err := installCertOperator(ctx, config)
 		if err != nil {
 			return microerror.Mask(err)
 		}
+	}
 
-		err = config.Release.InstallOperator(ctx, key.CertOperatorReleaseName(), release.NewStableVersion(), values, corev1alpha1.NewCertConfigCRD())
+	{
+		err := installNodeOperator(ctx, config)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
 	{
-		err := ensureCertConfigsInstalled(ctx, env.ClusterID(), config)
+		err := createGSReleaseContainingOperatorVersion(ctx, config)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
+	return nil
+}
+
+func installCertOperator(ctx context.Context, config Config) error {
+	certOperatorValues := `Installation:
+  V1:
+    Auth:
+      Vault:
+        Address: http://vault.default.svc.cluster.local:8200
+        Host: ""
+        CA:
+          TTL: 1440h
+        Certificate:
+          TTL: ""
+        Token:
+          TTL: ""
+        Version: ""
+    GiantSwarm:
+      CertOperator:
+        CRD:
+          LabelSelector: ""
+    Guest:
+      Calico:
+        CIDR: ""
+        Subnet: ""
+      Docker:
+        CIDR: ""
+      IPAM:
+        CIDRMask: ""
+        NetworkCIDR: ""
+        PrivateSubnetMask: ""
+        PublicSubnetMask: ""
+      Kubernetes:
+        API:
+          Auth:
+            Provider:
+              OIDC:
+                ClientID: ""
+                IssuerURL: ""
+                UsernameClaim: ""
+                GroupsClaim: ""
+          ClusterIPRange: ""
+          EndpointBase: k8s.%s
+        ClusterDomain: ""
+      SSH:
+        UserList: ""
+    Provider:
+      AWS:
+        Route53:
+          Enabled: false
+        S3AccessLogsExpiration: 0
+        TrustedAdvisor:
+          Enabled: false
+      Kind: ""
+    Secret:
+      CertOperator:
+        Service:
+          Vault:
+            Config:
+              Token: %s
+    Security:
+      RestrictAccess:
+        GuestAPI:
+          Public: false
+`
+	err := installLatestReleaseChartPackage(ctx, config, "cert-operator", fmt.Sprintf(certOperatorValues, env.CommonDomain(), env.VaultToken()))
```

Comment:
I'm not happy with the Sprintf here. I smell bugs. Why not templating? (I know this is only moved)